### ISSUE TITLE
How to test my own point clouds?

### ISSUE THREAD

**Issue by Eusford** [SimingYan/HPNet#16]
Hi Siming, I'm trying to build upon your code and want to predict my own point clouds *.ply files. My data provide points and normals, can you please tell me how to make valid inputs?

**Comment by SimingYan** [comment]
Hi Eusford,

Thanks for your interest! My suggestion is you can take a look at this file `dataloader/ABCDataset.py` about how to build the dataloader. But if you only have points and normals, it might be hard to obtain the ground truth instance label and primitive type label..

**Comment by Eusford** [comment]
Ok, I'll do some attempts first! 

And for the latter problem, my task is to identify possible primitives and fit their parameters with an input point cloud (just like a test). I wonder if your code can work without 'T_gt' and 'T_param'?

**Comment by SimingYan** [comment]
yes I think so! You can directly use our [pre-trained model](https://drive.google.com/file/d/1fj84kyD9CGT8j61IW-xSWZ5q4q5IpoYx/view) to test your point cloud data:)
